code/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Project Library</title>
+  </head>
+  <body>
+    <header>
+      <h1>Project Library</h1>
+    </header>
+    <div class="wrapper">
+      <div id="filter-books" class="filter-box">
+        <p class="headings-top">Filter books by genre</p>
+        <button id="all" class="filter-buttons">All</button>
+        <button id="fantasy" class="filter-buttons">Fantasy</button>
+        <button id="horror" class="filter-buttons">Horror</button>
+        <button id="adventure" class="filter-buttons">Adventure</button>
```

Comment:
no need to pass an `id` if you're not using it on your stylesheet, keep only valuable code 🧹

Reply:
We used the id for the DOM selectors in the js file, and the class for styling in CSS.